R/mod_tabpanel_data.R

```diff
@@ -72,6 +72,7 @@ mod_tabpanel_data_ui <- function(id) {
           shiny::uiOutput(ns("missing_vals"))
         ),
         hr(),
+        shiny::uiOutput(ns("negative_ages")),
```

Comment:
I like that in your message it is directly included what will happen with the negative ages. Maybe can you just adapt this for the case_id message as well that there it is written that these Case IDs are excluded from analysis. Then we can remove it in line 68/69

Reply:
Sure, I'll see what I can do about it.

Reply:
If not no worries. Just a minor thing

Reply:
Ok I just adjusted the code as you requested and pushed the changes. Please tell what you think of it :)

Reply:
Thanks, it is good. Have a nice and well deserved holiday!

Reply:
Thank you! :)